src/views/Result.vue

```diff
@@ -547,112 +343,108 @@ button:focus {
   margin: 0;
 }
 
-.incomplete-buttons {
-  border-radius: 24px;  
-  background-color: #FFFFFF;
-  color: #454F5B;
-  margin-right: 12px;
-  margin-bottom: 12px;
-  display: inline-block;
-  padding: 12px 20px;
-  text-decoration: none;
-}
-
 .incomplete-box-inner {
   padding-top: 28px;
   padding-left: 72px;
   padding-right: 72px;
 }
```

Comment:
Consider adding `text-align: center` here. This will fix center alignment for `.incomplete-box-desc`, which is left aligned at the moment.
If you do this, then you can remove `text-align: center` on `.button-wrapper` below.